src/classes/OPP_OpportunityNaming_TDTM.cls

```diff
@@ -0,0 +1,14 @@
+public class OPP_OpportunityNaming_TDTM extends TDTM_Runnable {
+
+    public override DmlWrapper run(List<SObject> newlist, List<SObject> oldlist, 
+        TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
+
+        DmlWrapper dmlWrapper = new DmlWrapper();
+
+        list<Opportunity> oppsForUpdate = OPP_OpportunityNaming.getOppNamesAfterInsert((list<Opportunity>) newlist);
```

Comment:
do you want to put this within an if for the correct triggerActions, rather than assuming only the correct ones are in our TDTM table?